generators/app/templates/docs/README.md

```diff
@@ -15,3 +15,14 @@ import <%= projectClassName %> from '<%= projectName %>';
 
 <<%= projectClassName %> {props...} />
 ```
+
+## Component Features
+
+<!-- Uncomment supported features.
```

Comment:
My concern is people forget to delete items and then it looks like a component features things it may not. I lean towards having people opt-in instead of opt-out of listing component supported features.